WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift

```diff
@@ -47,9 +67,13 @@ class LoginEpilogueViewController: UIViewController {
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         super.prepare(for: segue, sender: sender)
-        if let vc = segue.destination as? LoginEpilogueTableView {
-            tableViewController = vc
+
+        guard let epilogueTableViewController = segue.destination as? LoginEpilogueTableViewController, let credentials = credentials else {
+            return
```

Comment:
Even though its currently wired up where `credentials` will be set *before* `prepare` is called, there's nothing that enforces that.

It might be enough to add a note clarifying this caveat to our future selves, but would it be worth making a tweak so `credentials` isn't required here?  I'm just thinking about how this could be future proofed.

Its interesting that `viewDidLoad` requires `credentials` to be set and yields a `fatalError` if they aren't.  Should the way `credentials` are treated in these two methods be consistent?

Reply:
Personally, i don't like the entire mechanism. I would have probably implemented both `LoginEpilogueViewController` <> `LoginEpilogueTableViewController` into a single entity (trying not refactor _eeeeverything_ right right now, but i was tempted!!!)

Let's add a fatalError() if `credentials = nil`, so that it's consistent. Good call Eric!!